### ISSUE TITLE
<...> is not a function error (v 15.0.2)

### ISSUE THREAD

**Issue by danieldegtyarev** [bridgedotnet/Bridge#1856]
I did simplified code (see below) to show the issue, it generates

```javscript
this.children.getChanged().Demo$IObservable$1$Demo$CollectionChangeArgs$1$Demo$HtmlRenderElement$observe(Bridge.fn.bind(this, this.onChildrenChanged));
```
And you can see an error 

```js
this.children.getChanged(...).Demo$IObservable$1$Demo$CollectionChangeArgs$1$Demo$HtmlRenderElement$observe is not a function
```

### Steps To Reproduce

```csharp
public interface IObservable<T>
{
	void Observe(Action<T> args);
}

public class Signal<T> : IObservable<T>{
   public Signal(string x){}
   
   public void Observe(Action<T> a){}
}

public class CollectionChangeArgs<T>{
    
}

public class Collection<T> 
{
        Signal<CollectionChangeArgs<T>> changed;

        public IObservable<CollectionChangeArgs<T>> Changed
        {
            get { return changed ?? (changed = new Signal<CollectionChangeArgs<T>>("Collection<T>.Changed")); }
        }
        
        public void Foo(){}
} 

public class HtmlRenderElement{
Collection<HtmlRenderElement> children;
		public Collection<HtmlRenderElement> Children        
        {
            get
            {
                if (children == null)
                {
                    children = new Collection<HtmlRenderElement>();
                    children.Changed.Observe(OnChildrenChanged);
                }
                return children;
            }
        }
        
        void OnChildrenChanged(CollectionChangeArgs<HtmlRenderElement> a){
        
        }
}

public class Program
{
    public static void Main()
    {
        var x = new HtmlRenderElement();
        x.Children.Foo();
        Console.WriteLine("Hello World!");
    }
}

```

(Eh, kinda blocker for me)

**Comment by vladsch** [comment]
Thanks for the report, the issues is already fixed and the fix will be included to upcoming 15.1 release

**Comment by danieldegtyarev** [comment]
Excelent, thank you!
